rpcs3/Emu/Cell/Modules/sceNpTrophy.cpp

```diff
@@ -803,7 +801,7 @@ error_code sceNpTrophyGetTrophyIcon(u32 context, u32 handle, s32 trophyId, vm::p
 		return hidden ? SCE_NP_TROPHY_ERROR_HIDDEN : SCE_NP_TROPHY_ERROR_LOCKED;
 	}
 
-	fs::file icon_file(vfs::get("/dev_hdd0/home/00000001/trophy/" + ctxt->trp_name + fmt::format("/TROP%03d.PNG", trophyId)));
+	fs::file icon_file(vfs::get(fmt::format("/dev_hdd0/home/%s/trophy/%s/TROP%03d.PNG", Emu.GetUsr(), ctxt->trp_name, trophyId)));
```

Comment:
use concat instead

Reply:
This one I'm gonna revert to the original way that uses fmt::format only for the trophyId. I think that format specifier is necessary for the PNG name, unless you know of a better method.